apps/wallet/src/test-utils/vault.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fromExportedKeypair } from '@mysten/sui.js';
+
+import { toEntropy } from '_src/shared/utils/bip39';
+
+import type { Keypair } from '@mysten/sui.js';
+
+export const testMnemonic =
+    'loud eye weather change muffin brisk episode dance mirror smart image energy';
+export const testEntropySerialized = '842a27e29319123892f9ba8d9991c525';
+export const testEntropy = toEntropy(testEntropySerialized);
+export const testEd25519Serialized = Object.freeze({
+    schema: 'ED25519',
+    privateKey:
+        'a3R0jvXpEziZLHsbX1DogdyGm8AK87HScEK+JJHwaV99nEpOfYblbYS3ci9wP2DT5YZtE3e4v/HBsN39kRz60A==',
+} as const);
+export const testEd25519 = fromExportedKeypair(testEd25519Serialized);
+export const testEd25519Address = testEd25519.getPublicKey().toSuiAddress();
+export const testSecp256k1Serialized = Object.freeze({
+    schema: 'Secp256k1',
+    privateKey: '4DD3CUtZvbc9Ur69tTvKaLeIDptxNa9qZcpkyXWjVGY=',
+} as const);
+export const testSecp256k1 = fromExportedKeypair(testSecp256k1Serialized);
+export const testSecp256k1Address = testSecp256k1.getPublicKey().toSuiAddress();
+type TestDataVault = typeof testVault;
+/**
+ * A test vault with 2 keypairs
+ */
+export const testVault = Object.freeze({
```

Comment:
Is there a way to generate this instead of having the serialized JSON in here like this? Seems like a lot of work to keep up-to-date in the event we change any of our storage.

Reply:
Maybe the name is confusing, this contains test data to easily have access to them. The actual stored value (and what can change from the app) is the encrypted part. Which probably can be generated somehow but if we change the storage (add extra version) probably that would have to change as well

Reply:
Yeah it's tough to figure the right way to do this. I don't love having serialized JSON here because I think it's highly unintuitive how to generate it to make tests pass. An idea for the future:
- For most tests just assume that the vault generation logic works and use that to generate a vault JSON we can use. This depends on the vault logic to work correctly (so we'll be using what we're testing), but that's probably fine so long as we have assertions here + we can have good messaging when it fails because it's fairly logically separated (e.g. vault creation can fail separately from test run).
- We still would want some static JSON for tests with old versions. For these we could write a script to spin out a vault and snapshot it to some `fixtures/vault_v1.json` file, so that it's at least automated.

We don't need to do this now, I can create a follow-up issue.